lab01-voronin-roman

Lisoon22 opened the pull request.

Lisoon22 committed:
lab01: implement user authentication

- add data validation
- make flutter app with counter, personal info, and registration form

Lisoon22 committed:
lab01: implement user authentication

- add data validation
- make flutter app with counter, personal info, and registration form

Lisoon22 committed:
lab01: implement calculator, taskmanager, and user classes, build the flutter app with counter, registation form, and profile

Lisoon22 committed:
lab01-voronin-roman

lab01: implement calculator, taskmanager, and user classes, build the…

Lisoon22 committed:
lab02: implement chat backend, message managing, chat app with user profile

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

Lisoon22 committed:
Merge pull request #2 from Lisoon22/lab02-voronin-roman

lab02: implement chat backend, message managing, chat app with user p…

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

acecution commented:
Great implementation! One small suggestion: Consider following CONTRIBUTING.md stardarts for writing information for PR.

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

Lisoon22 committed:
lab05: JWT authentication and validation

- Add jwt service
- Implement authentication
- Add JWT token validation middleware
- Include comprehensive unit tests
- Update documentation

Lisoon22 committed:
Merge branch 'timur-harin:main' into main

Lisoon22 committed:
lab07initial

Lisoon22 committed:
lab07: docker and containers

- Make a container from backend, write the deploy.yml

Lisoon22 committed:
Merge pull request #6 from Lisoon22/lab07-voronin-roman

Lab07: Docker

Lisoon22 committed:
Add web/index.html for Flutter web build

Lisoon22 committed:
Add web/index.html for Flutter web build

Lisoon22 committed:
Add web/index.html for Flutter web build

Lisoon22 committed:
Add web/index.html for Flutter web build444

Lisoon22 committed:
Add web/index.html for Flutter web build444

Lisoon22 committed:
Merge pull request #8 from Lisoon22/lab07-voronin-roman3

Lab07 voronin roman3
